extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -57,6 +57,11 @@ class AmpStickyAd extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    userAssert(
+      !AMP.stickyAdLoaded,
```

Comment:
Why does this need to be a stored boolean? Can't we just querySelector?

Reply:
Intention is to ensure exactly one slot is rendered, but yeah we can querySelector that as well.